borg/cache.py

```diff
@@ -390,6 +400,39 @@ def legacy_cleanup():
             self.do_cache = os.path.isdir(archive_path)
             self.chunks = create_master_idx(self.chunks)
 
+    def check_cache_compatibility(self):
+        my_features = Manifest.SUPPORTED_REPO_FEATURES
+        if self.ignored_features & my_features:
+            return False
```

Comment:
I'm not really sure if I understand this correctly.

ignored_features is the set of features that are mandated for some operations, but not those the cache was originally built for, and these were not supported by the client building / last updating the cache.

So `ignored_features & my_features` would be the set of features that were *not* supported by the previous client. But why wipe the cache here?